File: SistemaBancario_java/src/ContaPoupanca.java

public class ContaPoupanca extends ContaSimples{
	
	public ContaPoupanca() {}
	
	public ContaPoupanca(int numero, String cliente){
		super(numero, cliente);
	}
	
	void acrescentaRendimento(double taxa){
		setSaldo(getSaldo() * (1 + taxa));
	}


}

File: SistemaBancario/src/ContaPoupanca.cpp
#include "ContaPoupanca.h"

ContaPoupanca::ContaPoupanca(int numero, string cliente) : ContaSimples(numero, cliente){

}

ContaPoupanca::~ContaPoupanca() {
	//std::cout << "'destruindo' poupanca\n";
}

void ContaPoupanca::acrescentaRendimento(double taxa){
	this->saldo *= 1 + taxa;
}
